ci/azure-pipelines.yml

```diff
@@ -170,6 +170,26 @@ stages:
             displayName: DKMS module build on Ubuntu
             condition: contains(variables['build_container'], 'ubuntu')
 
+          - bash: |
+              set -eEx
+              test -f Makefile && make -i distclean || true
+              sudo apt-get update
+              sudo apt-get install -y debhelper
+
+              ./autogen.sh
+              ./configure --disable-kernel-module
+              make dist-gzip
+              tarball="$(ls xpmem-*.tar.gz)"
+              tar zxf "$tarball"
+              cd "${tarball%.tar.gz}"
+              dpkg-buildpackage --build=binary -uc
+              cd -
+              dpkg --contents libxpmem_*.deb
+              dpkg --contents libxpmem-dev_*.deb
+              dpkg --contents libxpmem-dbgsym_*.ddeb
```

Comment:
is it worth to check the content in some way? (i.e. parse the output)

Reply:
pushed to try to improve tests